client/settings/payment-method-icon/style.scss

```diff
@@ -0,0 +1,13 @@
+.woocommerce-payments__payment-method-icon {
+	display: inline-flex;
+	align-items: center;
+	vertical-align: middle;
+
+	@media screen and ( max-width: 425px ) {
+		max-width: 160px;
+	}
+
+	img {
+		margin-right: 4px;
```

Comment:
When the `showName` is falsy, the margin seems to still be applied, which could cause some spacing issues - is the `showName` prop needed?

Reply:
I was thinking this could replace other icons in the app, as a way to "React-ify" that part, that's way I came out with that prop. I'll fix the margin issue for this case :+1: